charts/stable/nginx-proxy-manager/values.yaml

```diff
@@ -75,6 +75,8 @@ workload:
                 exec:
                   command:
                     - /bin/check-health
+                  initialDelaySeconds: 300
```

Comment:
This will make it a total of 10mins,

https://github.com/truecharts/library-charts/blob/9fed4abf501782ed95335eb2d7d31094a85e1424/library/common/values.yaml#L55-L61

Startup already ignores failures for 5min.
I don't think I can accept this change.

1. With our minimum hardware requirements you shouldnt have any issues.
2. Upstreams might have to rethink their startup scripts. chowing a bunch of files on every restart on a **container** is not ok.